sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f

```diff
@@ -307,44 +279,38 @@ SUBROUTINE CALRAD_WCLOUD
 
   !     DO NOT FORGET TO ADD YOUR NEW IGET HERE (IF YOU'VE ADDED ONE)      
   !     START SUBROUTINE CALRAD.
-  ifactive: if (iget(327) > 0 .or. iget(328) > 0 .or. iget(329) > 0       &
```

Comment:
@gspetro-NOAA These IDs and related configuration should be kept for duplicating ops. applications.